rpm/SPECS/contextBroker.spec

```
@@ -72,7 +72,7 @@ fi
 
 %pre
 getent group %{owner} >/dev/null || groupadd -r %{owner}
-getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /opt/orion -s /bin/bash -c 'Orion account' %{owner}
+getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /usr/share/contextBroker/ -s /bin/bash -c 'Orion account' %{owner}
```

Better to use:

```suggestion
getent passwd %{owner} >/dev/null || useradd -r -g %{owner} -m -d /home/orion/ -s /bin/bash -c 'Orion account' %{owner}
```

CC: @cesarjorgemartinez

---

Fixed in https://github.com/telefonicaid/fiware-orion/pull/3863/commits/8ba3fec7bdd26793b70ad14d9ffcd6bcc33520af